core/server/services/members/middleware.js

```diff
@@ -1,8 +1,29 @@
 const common = require('../../lib/common');
+const constants = require('../../lib/constants');
 const shared = require('../../web/shared');
 const labsService = require('../labs');
 const membersService = require('./index');
 
+module.exports.public = (siteApp) => {
```

Comment:
I think this works living here - kinda? I'm not too sure tbh - but yeah it is annoying that the file is in `public` - HMMMMM. 

I guess you can think of the file as either being part of members, or just a frontend script which integrates w/ members :thinking:

Reply:
Definitely a weird one :) The reason why I combined it into this module because it also uses the check if members are enabled `shared.middlewares.labs.members`, so _kinda_ belongs in one bag with the rest. Otherwise, it could live in the site/app.js module.